src/SUMMARY.md

```diff
@@ -1,27 +1,27 @@
 # Summary
 
-[CHERI Exercises and Missions](cover/README.md)
+[🍒 CHERI Exercises and Missions](cover/README.md)
 - [Introduction](introduction/README.md)
     - [Background reading](introduction/background.md)
-    - [Obtaining CHERI Compilers and Simulators](introduction/get.md)
+    - [🍒 Obtaining CHERI Compilers and Simulators](introduction/get.md)
     - [Checking out the Exercises Themselves](introduction/get-us.md)
     - [Cross compilation](introduction/cross.md)
-        - [Using the CHERI Software Release](introduction/cross-ccc-release.md)
+        - [🍒 Using the CHERI Software Release](introduction/cross-ccc-release.md)
         - [Helper script](introduction/cross-ccc.md)
         - [Manual execution](introduction/cross-diy.md)
     - [Running CHERI Programs](introduction/run.md)
         - [Using QEMU](introduction/run-qemu.md)
             - [Using QEMU with the CHERI Software Release](introduction/run-qemu-release.md)
-            - [Using SMB under QEMU](introduction/run-qemu-smb.md)
+            - [🍒 Using SMB under QEMU](introduction/run-qemu-smb.md)
         - [Using Morello FVP](introduction/run-fvp.md)
-        - [Using SSH With The Emulated System](introduction/run-ssh.md)
+        - [🍒 Using SSH With The Emulated System](introduction/run-ssh.md)
         - [Rebuilding the Disk Image](introduction/run-build-disk.md)
     - [Inspecting CHERI Programs](introduction/inspect.md)
         - [Disassembly](introduction/inspect-disassemble.md)
         - [Debugging](introduction/inspect-debug.md)
-- [ASPLOS'22 Exercises]()
-    - [Compile and run baseline and CHERI-enabled programs](exercises/compile-and-run/README.md)
+    - [🍒 Compile and run baseline and CHERI-enabled programs](exercises/compile-and-run/README.md)
         - [Answers](exercises/compile-and-run/answers.md)
+- [🍒 ASPLOS'22 Exercises]()
```

Comment:
I think I'd rather leave the 🍒 off this one and leave the `compile and run` below this (but with its newly added 🍒).  While they're welcome to read ahead, we probably don't want to appear that we're insisting on it.  At least some of the existing prose refers to compile-and-run as "the first exercise"... I guess we could change that.